### PR TITLE
fix loot of mobs

### PR DESCRIPTION
**Pull request by adoxentor**
my first public fix

# Changes proposed in this pull request:
-added tome as a rare drop, more common on player kills
-fixed drops of barbarians/pirates

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/event/LootTableEventHandler.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.minecolonies.api.util.constant.Constants.MOD_ID;
+
+public class LootTableEventHandler {
+    public static List<String> mobs = Arrays.asList(
+            "minecraft:entities/skeleton",
```
Comment (Asherslab) [inline src/main/java/com/minecolonies/coremod/event/LootTableEventHandler.java]: We do not wish for the tome to be dropped by any entities besides the raiders, it adds a level of rarity to them, as well as requiring the player to endure these raids

**Comment by Raycoms** [comment]
thanks for the contribution, may you please sign the CLA?

**Comment by adoxentor** [comment]
> thanks for the contribution, may you please sign the CLA?

I did

**Comment by Raycoms** [comment]
Thanks =)

**Pull request merged**